src/components/App.js

```diff
@@ -11,17 +11,16 @@ import Event from './event/Event'
 import Loading from './common/Loading';
 import NotFound from './error/NotFound';
 import {useApp} from '../context/AppContext'
+import {useAuth} from '../context/AuthContext'
 
 
 function App() {
   const { appLoading, appError } = useApp();
-  
+  const { currentUser } = useAuth()
```

Comment:
This line keeps breaking my brain. I'm not sure how the `currentUser` is coming out of `useAuth()` instead of `AuthProvider`